cms/djangoapps/contentstore/views/course.py

```diff
@@ -505,7 +505,11 @@ def course_listing(request):
     optimization_enabled = GlobalStaff().has_user(request.user) and \
         WaffleSwitchNamespace(name=WAFFLE_NAMESPACE).is_enabled(u'enable_global_staff_optimization')
 
-    org = request.GET.get('org', '') if optimization_enabled else None
+    organizations = get_organizations() if optimization_enabled else []
+    org_names_list = [(org["short_name"]) for org in organizations]
+
+    selected_org = request.GET.get('org', '')
+    org = selected_org if optimization_enabled and selected_org != '' else None
```

Comment:
@elvinquero  What do you think about this?
```suggestion
    selected_org = request.GET.get('org', '')
    org = selected_org if optimization_enabled and selected_org else None
```
Is there a reason to validate that selected_org isn't empty?